src/gui/statusbar.cpp

```diff
@@ -50,8 +50,6 @@
 StatusBar::StatusBar(QStatusBar *bar)
     : m_bar(bar)
 {
-    qApp->setStyleSheet("QStatusBar::item { border-width: 0; }");
```

Comment:
I just did the move in e35a7ef9d3421. Placing here is more intuitive IMO, or, this is blocking this PR?